sdk/nodejs/cmd/run/run.ts

```diff
@@ -270,6 +270,11 @@ ${defaultMessage}`);
         });
     }
 
+    const checkForTSAndJSModules = async () => {
```

Comment:
I wasn't sure if it would be worth passing `program` and `typescript` as parameters here, but the general pattern in this module seems to avoid that.